content/show/self-hosted/_index.md

```diff
@@ -6,7 +6,7 @@ draft = false
 categories = ["Self-Hosted"]
 show = "self-hosted"
 hosts = ["chris","alex"]
-
+sponsors = ["cloudfree.shop-ssh"]
```

Comment:
Since we already have the cloudfree.shop-ssh file, @ChanceM was able to re-use that here. We can manually add them though as well, similar to how we can add [network wide sponsors](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/451/files#r996163919).

This was made possible because of these lines:
https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/451/files#diff-fe844ea8500b78927605d6141fba212f543102c5504e26207af582154ad609e1R19-R23